internal/report/snapshot/snapshot_buffers.go

```diff
@@ -258,7 +258,7 @@ func (l *TraceBuffer) Add(td ptrace.Traces) {
 }
 
 // ConstructPayload condenses the buffer and serializes to protobuf
-func (l *TraceBuffer) ConstructPayload() ([]byte, error) {
+func (l *TraceBuffer) ConstructPayload(searchQuery *string, minimumTimestamp *time.Time) ([]byte, error) {
```

Comment:
nit: We changed this in the logging PR but the parameters `searchQuery` and `minimumTimestamp` differ from `query` and `minTimestamp` used in the `filter_traces.go` file. I forget what Brandon standardized, but we should make them all the same.